docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -2,396 +2,1398 @@
  "cells": [
   {
    "cell_type": "markdown",
+   "id": "ea8f6c9f",
    "metadata": {},
    "source": [
     "# Model monitoring and drift detection\n",
     "\n",
-    "This tutorial illustrates leveraging the model monitoring capabilities of MLRun to deploy a model to a live endpoint and calculate data drift.\n",
+    "This tutorial illustrates the basic model monitoring capabilities of MLRun: deploying a model to a live endpoint and \n",
+    "calculating data drift.\n",
```

Comment:
The flow here needs to be better explained, it starts with real-time and then goes to batch, we should better explain to the user what they are expected to see and the general idea of the flow

Reply:
added more explanation